.github/workflows/pr.yaml

```diff
@@ -0,0 +1,232 @@
+          - 3306
+        env:
+          MYSQL_USER: db
+          MYSQL_PASSWORD: db
+          MYSQL_DATABASE: db_test
+          MYSQL_ROOT_PASSWORD: password
+        options: --health-cmd="mysqladmin ping" --health-interval=5s --health-timeout=2s --health-retries=3
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ["8.1"]
+    name: PHP Unit tests (PHP ${{ matrix.php }})
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php}}
+          extensions: apcu, ctype, iconv, imagick, json, redis, soap, xmlreader, zip
+          coverage: none
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-composer-
+
+      - name: Install Dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+
+      - name: PHP Unit
+        env:
+          PORT: ${{ job.services.mariadb.ports[3306] }}
+        run: DATABASE_URL="mysql://db:db@127.0.0.1:$PORT/db_test?serverVersion=mariadb-10.5.13" composer run tests
+
+  apispec:
+    runs-on: ubuntu-20.04
+    name: API Specification validation
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ["8.1"]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Setup PHP, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php}}
+          extensions: apcu, ctype, iconv, imagick, json, redis, soap, xmlreader, zip
+          coverage: none
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ matrix.php }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ matrix.php }}-composer-
+
+      - name: Install Dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
+
+      - name: Export specifications (yaml)
+        run: bin/console api:openapi:export --yaml --output=public/api-spec-v1.yaml --no-interaction
+
+      - name: Check for changes in specifications (yaml)
+        run: git diff --diff-filter=ACMRT --exit-code public/api-spec-v1.yaml
+
+      - name: Export specifications (json)
```

Comment:
We have no need for the json export in this project the yaml is fine for validation.